src/BlockID.cpp

```diff
@@ -190,7 +190,7 @@ class cBlockIDMap
 
 
 bool cBlockIDMap::m_bHasRunInit = false;
-static cBlockIDMap gsBlockIDMap;
+static cBlockIDMap g_gsBlockIDMap;
```

Comment:
This is why you can't just blindly pass `-fix` and not check the output.

Reply:
Yeah, I thought, that I've run `grep g_g`...

Reply:
You've got lucky ;D. Just 2/142 files contained it...